src/core/services/theming/theming.spec.js

```diff
@@ -612,6 +612,51 @@ describe('$mdThemeProvider with disabled themes', function() {
   });
 });
 
+describe('$mdThemeProvider with custom styles that include nested rules', function() {
+  it('appends the custom styles taking into account nesting', function() {
+    module('material.core', function($mdThemingProvider) {
+      $mdThemingProvider.generateThemesOnDemand(false);
+      var styles = `
```

Comment:
Can you switch this to be a regular ES5 string? We're not using ES6 yet. This is mainly for consistency.